app/config/config.yml

```diff
@@ -11,6 +11,13 @@ imports:
 
 # Basic configuration for the Symfony framework features
 framework:
+    # Uncomment the 'ide' option to turn file paths into clickable links that
+    # open the given file path and line number using your favorite IDE.
+    # Supported values are 'textmate', 'macvim', 'emacs' and 'sublime'
+    # See http://symfony.com/doc/current/reference/configuration/framework.html#ide
+    #
+    # ide: sublime
```

Comment:
@stof why we need to set this value into all environment as I think it's useful only in dev.